venv/Juxtaposition.py

```diff
@@ -0,0 +1,9 @@
+import sys
```

Comment:
You can delete this file. It's duplicate of the file in Kattis/ folder.
You can do so by navigate to Kattis, and then remove:
```
cd Kattis
rm -f Juxtaposition.py
```

Reply:
I deleted the venv file so now there is only Kattis left.